body-tracking-samples/simple_3d_viewer/main.cpp

```diff
@@ -99,24 +103,172 @@ bool ParseInputSettingsFromArg(int argc, char** argv, InputSettings& inputSettin
         {
             inputSettings.CpuOnlyMode = true;
         }
+        else if (inputArg == std::string("OFFLINE"))
+        {
+            inputSettings.Offline = true;
+        }
+     
         else
         {
-            return false;
+            inputSettings.FileName = inputArg;
         }
+     
     }
     return true;
+
 }
 
-int main(int argc, char** argv)
-{
-    InputSettings inputSettings;
-    if (!ParseInputSettingsFromArg(argc, argv, inputSettings))
+
+void playFile(InputSettings inputSettings) {
```

Comment:
Nit: Please capitalize "P" to make the naming consistent.

Reply:
done